### DISCUSSION TITLE
Add `fields` parameter shorthand

### DISCUSSION THREAD

**Discussion by raflymln** [directus/directus#14238] [Feature Requests]
Yesterday i was trying to fetch the api, and i got this example data:

```json
"data": {
  "translations": [
      {
          "id": 1,
          "faqs_id": 1,
          "languages_code": "en",
          "question": "Are there any hidden fees?",
          "answer": "No, we don't charge any fees for using our services. We only charge for the services you use."
      },
  ]
}
```

The thing is, what i'm trying to get was only the **question** and the **answer** inside translations data, and for that, i need to set the global fields parameters like this:

```
http://example.com/items/collectionName?fields=translations.question,translations.answer
```

I mean, that's still looks great, but what if we need more parameters, that would result in long url right? So it would be great in the future if you guys can implement something like this

```
http://example.com/items/collectionName?fields=translations.{question,answer}
```

In this case, the query would be decoded as `translations.question` and `translations.answer`.

Hope this improvement can be found the solution and would be applied as soon as possible since this would help a lot in development.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
